spec/spec_helper.rb

```diff
@@ -11,6 +11,20 @@
 
   config.shared_context_metadata_behavior = :apply_to_host_groups
 
+  require "webmock/rspec"
```

Comment:
Ignoring this for now, because I am getting conflicting messages from Hound and Rubocop about `Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.`